PetstoreAPI/Settings.cs

```diff
@@ -7,6 +7,4 @@ class Settings
     {
         public string WebsiteUrl { get; set; }
```

Comment:
Is this only used for the URL of the website? Or the full path to the JSON you're downloading?